tests/system/aiplatform/test_featurestore.py

```diff
@@ -247,3 +249,19 @@ def test_search_features(self, shared_state):
         assert (
             len(list_searched_features) - shared_state["base_list_searched_features"]
         ) == 6
+
+    def test_online_reads(self, shared_state):
+        assert shared_state["user_entity_type"]
+        assert shared_state["movie_entity_type"]
+
+        user_entity_type = shared_state["user_entity_type"]
+        movie_entity_type = shared_state["movie_entity_type"]
+
+        user_entity_views = user_entity_type.read(entity_ids="alice")
+        assert type(user_entity_views) == pd.DataFrame
```

Comment:
Can we test the actual values?

Reply:
there is a check actual value a couple lines below, also added another unit test for checking data frame construction, where it has more cases